cmd/snap-confine/ns-support.c

```diff
@@ -501,9 +531,10 @@ void sc_create_or_join_ns_group(struct sc_ns_group *group,
 		}
 		group->should_populate = true;
 	}
+	return 0;
 }
 
-bool sc_should_populate_ns_group(struct sc_ns_group *group)
+bool sc_should_populate_ns_group(struct sc_ns_group * group)
```

Comment:
Oddly this is coming out of my machine's `make fmt`. If indent changed rules again I will call for a flag day and propose that we move to `clang-format` that _at least_ has a very readable and sane output without 80 column constraint.

Reply:
/me happens to like the 80 column constraint. ;)